README.md

```diff
@@ -18,6 +18,7 @@ use std::error::Error;
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
     let client = Client::builder()
+        .token("MY TOKEN")
```

Comment:
In this example please use `std::env`; we historically discourage hardcoding in documentation.